azure-maven-plugin-lib/src/main/java/com/microsoft/azure/maven/artifacthandler/FTPArtifactHandlerImpl.java

```diff
@@ -6,24 +6,26 @@
 
 package com.microsoft.azure.maven.artifacthandler;
 
+import com.microsoft.azure.management.appservice.FunctionApp;
 import com.microsoft.azure.management.appservice.PublishingProfile;
+import com.microsoft.azure.management.appservice.WebApp;
 import com.microsoft.azure.maven.AbstractAppServiceMojo;
 import com.microsoft.azure.maven.FTPUploader;
 import com.microsoft.azure.maven.Utils;
 import com.microsoft.azure.maven.deploytarget.DeployTarget;
 import org.apache.maven.model.Resource;
 import org.apache.maven.plugin.MojoExecutionException;
 
+import java.io.File;
 import java.io.IOException;
 import java.nio.file.Paths;
 import java.util.List;
 
 public class FTPArtifactHandlerImpl<T extends AbstractAppServiceMojo> implements ArtifactHandler {
     private static final String DEFAULT_WEBAPP_ROOT = "/site/wwwroot";
     private static final int DEFAULT_MAX_RETRY_TIMES = 3;
-    private static final String NO_RESOURCES_CONFIG = "No resources specified in pom.xml. Skip artifacts deployment.";
-    private static final String WEBAPP_PLUGIN_NAME = "azure-webapp-maven-plugin";
     private static final String MAVEN_PLUGIN_POSTFIX = "-maven-plugin";
+    private static final String NO_RESOURCES = "No resources were specified in pom.xml or copied to staging directory.";
```

Comment:
Better for error log here?

Reply:
https://github.com/Microsoft/azure-maven-plugins/issues/297

Opened a separate issue to discuss and solve it.